docs/site/Fields-filter.md

```diff
@@ -21,9 +21,16 @@ Where:
 - `<true|false>` signifies either `true` or `false` Boolean literal. Use `true`
   to include the property or `false` to exclude it from results.
 
+also can be used an array of strings with the properties
+
+<pre>
+{ fields: [<i>propertyName</i>, <i>propertyName</i>, ... ] }
+</pre>
+
 By default, queries return all model properties in results. However, if you
-specify at least one fields filter with a value of `true`, then by default the
-query will include **only** those you specifically include with filters.
+specify at least one fields filter with a value of `true` or put it in the
+array, then by default the query will include **only** those you specifically
+include with filters.
```

Comment:
Suggestion for r-structuring the initial sections to this:

```
A _fields_ filter specifies properties (fields) to include or exclude from the
results.

### Node.js API

#### `fields` as an object

<pre>
{ fields: {<i>propertyName</i>: <true|false>, <i>propertyName</i>: <true|false>, ... } }
</pre>

Where:

- _propertyName_ is the name of the property (field) to include or exclude.
- `<true|false>` signifies either `true` or `false` Boolean literal. Use `true`
  to include the property or `false` to exclude it from results.

By default, queries return all model properties in results. However, if you
specify at least one fields filter with a value of `true`, then by default the
query will include **only** those you specifically include with filters.


#### `fields` as an array


<pre>
{ fields: [propertyName1, propertyName2, ...] }
</pre>

Only the properties specified in the list will be returned.
```

@bajtos and others thoughts?

Reply:
The suggested structure looks good to me.

Just note that we support array + object format also for REST API, not just in Node.js (TypeScript).